getting-started.md

```diff
@@ -74,6 +75,20 @@ hs.hotkey.bind({"cmd", "alt", "ctrl"}, "W", function()
 end)
 ```
 
+### <a name="spoons"></a>Spoons
+
+Spoons are pre made plugins for Hammerspoon.
```

Comment:
```suggestion
Spoons are premade plugins for Hammerspoon.
```

Reply:
According to Apple's Dictionary, which taps into Oxford English dictionary for British English it's `pre-made`.

Web version of Oxford English redirects from `premade` to [pre-make](https://en.oxforddictionaries.com/definition/pre-make).

According to [Merriam Webster dictionary](https://www.merriam-webster.com/dictionary/premade), it's `premade`.

Oh, the dilema, which dictionary is more authoritative ?

Are you sticking with your original option of a single word, no hyphen? :)

Reply:
I'm happy with a hyphen or nothing. I'm not happy with a space :)

Reply:
Went with the hyphenated version